index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="EN">
+  <head>
+    <meta charset="utf-8">
+    <title>My Portfolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <h2>Welcome</h2>
+      <a title="Button"></a>
+    </header>
+    <section class="headline">
+      <h1>Hey there. I'm Amaury</h1>
+      <h1>I'm a software developer</h1>
+      <p> 
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <div>
+        <a href="#"><img src="images/icons/Linkedin-icon.svg" alt="LinkedIn icon"/></a>
+        <a href="#"><img src="images/icons/github.svg" alt="GitHub icon"/></a>
```

Comment:
[REQURIED] Please The list of icons should use a `ul` tag to group them, and `li` per icon.
semantically the icons group  is a list so it should be in ul elmenet